test/passing/quoted_strings.ml

```diff
@@ -0,0 +1,54 @@
+let foo = {%foo | foooooooooooooo |}
+
+let foo = (* A *) ({%foo | foooooooooooooo |} (* B *) [@attr]) (* C *)
+
+let foo = (* A *) {%foo sep| foooooooooooooo |sep}
+
+let foo = {%foo | foooooooooooooo |} [@@attr]
```

Comment:
I think it's expected that there's no space after the extension, like `{%foo|payload|}`. That's how they are described in the PR and in the manual, and that's consistent with delimiters.

Reply:
Oh yeah that's right, it's fixed now.